references/V1_OutboundCallback.md

```diff
@@ -210,6 +210,25 @@ This event is only sent to webhooks that
 |**type**  <br>*required*|The event type is `message-deleted`.|string|
 
 
+<a name="messageeditedbody"></a>
+### MessageEditedBody
+Notifies that an administrator has updated the contents of a message in a space.
```

Comment:
How about removing why/who the message has been edited and just leave it as:

"Notifies the subscribed app that a message has been edited"

Reply:
Addressed in commit 76554ff